cypress/e2e/hub/collections-detail.cy.ts

```diff
@@ -118,6 +118,7 @@ describe('Collections Details', () => {
       'contain',
       `/collections/validated/${namespace.name}/${collectionName}/details?version=1.0.0`
     );
+    cy.wait(1000);
```

Comment:
@RedOrNot we should avoid to hard code wait values. Could we assert that certain value was element should be present?